src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -422,6 +422,14 @@ public Boolean isHasRestrictedFiles(){
         }
     }
     
+    public boolean getHasRestrictedFiles(){
```

Comment:
I dont think you need this - being a boolean it should call the isHasRastrictedFiles method. But see my other note for if we even need the ui:hidden field.